.github/workflows/test-e2e.yaml

```diff
@@ -0,0 +1,65 @@
+name: Website Tests (Cypress)
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - website/**
+  pull_request:
+    paths:
+      - website/**
```

Comment:
Even though this is an e2e test, I think we're okay just running when `website/**` changes. Interface changes in the backend should be caught by the light weight contract tests. Just calling it out because I think what you have makes sense. 

(Though I'm biased because I wrote the contract tests ;) )

Reply:
Hmmm, now you've made me think about it again 😅. The main reason for it is so that we ignore untested changes in the bot or docs and such. I would probably be in favour of triggering on backend changes, as a kind of smoke test it exists to catch the case that you never expect to fail, including incompatible changes deeper in the backend than just the interface.

Reply:
You have a good point. It would be safer to run them for back end changes as well, just in case. A smoke test has value the contract test doesn't provide. It just stinks that it takes so long to run

Reply:
In the end all questions about when to trigger it really need to be decided by @yk or @andreaskoepf as it impacts the project fairly broadly. I realise now I should have called that out in the PR description.

Reply:
I think changes to `website` and `backend` make sense.